packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -46,6 +47,16 @@ const Link = forwardRef(
       }
     `);
 
+    // Supplied from LoggedInProvider
+    const { loggedIn } = useLoggedIn();
+
+    if (to.includes('one.newrelic.com')) {
+      // Using session storage, so the value will come as a string.
+      if (loggedIn === 'false') {
```

Comment:
i think the useLoggedIn hook always returns a boolean and not a string

Reply:
i was having a lot of trouble getting it to work just via a Boolean and idk why. i'll try again